docs/guide/integrations/nuxt3.md

```diff
@@ -0,0 +1,314 @@
+
+```sh
+npm install --save-dev @intlify/unplugin-vue-i18n
+```
+
+### Configure Nuxt config
+
+Configure `nuxt.config.ts` like the below:
+
+```diff
+ import { defineNuxtConfig } from 'nuxt'
++import { resolve, dirname } from 'node:path'
++import { fileURLToPath } from 'url'
++import VueI18nVitePlugin from '@intlify/unplugin-vue-i18n/vite'
+ 
+ // https://v3.nuxtjs.org/api/configuration/nuxt.config
+ export default defineNuxtConfig({
++  vite: {
++    resolve: {
++      alias: {
++        'vue-i18n': 'vue-i18n/dist/vue-i18n.runtime.esm-bundler.js'
++      }
++    },
++    plugins: [
++      VueI18nVitePlugin({
++        include: [
++          resolve(dirname(fileURLToPath(import.meta.url)), './locales/*.json')
++        ]
++      })
++    ]
++  }
+ })
+```
+
+The bundler for Nuxt 3 is vite by default. So we will use the `vite` option here for optimization.
+
+In `vite.resolve.alias`, set the `vue-i18n` alias to use only the Vue I18n runtime (`vue-i18n/dist/vue-i18n.runtime.esm-bundler.js`). This setting reduces the bundle size since the message compiler used by Vue I18n is not included.
+
+In `vite.plugins`, the plugin for `@intlify/unplugin-vue-i18n` is configured. As an option for this plugin, the `include` option specifies locale resources in json format placed in the `locales` directory. This allows `@intlify/unplugin-vue-i18n` to pre-compile locale resoruces at bundle time using Vue I18n message compiler internally. This improves the translation performance of Vue I18n and consequently the rendering performance of Nuxt 3 applications.
+
+### Inside of bundling with optimization
+
+When you have finished the setup, let’s run `npm run dev` to check it out!
+
+When you will access to `http://localhost:3000`, the behavior of the Nuxt 3 application remains the same, but there is a change in the banding of the Nuxt 3 application.
+
+The following is a comparison of bundle sizes measured in the network tab of devtools with and without `vite.resolve.alias`:
+
+![Reduce bundle size](/nuxt3-reduce-bundle-size.png)
+
+The area highlighted in blue is the code bundled by vite.
+
+`vite.resolve.alias` setting specifies a runtime-only module for Vue I18n, which reduces the bundle size.
+
+Also, you can see the changing in the bandling of locale resources.
+
+Code for locale resources, depending on whether or not the `@intlify/unplugin-vue-i18n` plugin to `vite.plugins` is set. Below:
+
+![Pre-compile](/nuxt3-pre-compile.png)
+
+Without the `@intlify/unplugin-vue-i18n` plugin to `vite.plugins`, locale resources are banded as **json**, but with this plugin set, locale resources are **converted from json to JavaScript code** for banding.
+
+Vue I18n just call the functions! if the locale resource is a function, since it has already been compiled.
+
+In this tutorial,  Nuxt 3 application is small, so we can not enough experience the performance of this optimization, but as the application gets larger, it will benefit from this optimization.
```

Comment:
```suggestion
In this tutorial,  the Nuxt 3 application is small, so we can not enough experience the performance of this optimization, but as the application gets larger, it will benefit from it.
```